### DISCUSSION TITLE
Totals to State Timeline

### DISCUSSION THREAD

**Discussion by Caseum96** [grafana/grafana#44193] [General]
Hey all,

At our office we monitor some machine for the on/off time. For this we use the State Timeline to display the values.
Is there a possibility to see all the total on and off time of the time range?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
